ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/clientlibs/js/checkoutform.js

```diff
@@ -180,7 +183,7 @@ function onSaveNDeliver() {
 			}
 
 			if (userDetails === 'Pin Number' && userDetailsValue) {
-				const pinRegexp = /^[0-9]{5}(?:-[0-9]{4})?$/;
+				const pinRegexp = /^[0-9]{6}(?:-[0-9]{4})?$/;
```

Comment:
@DeepaliAgrwl : was this regex change expected

Reply:
This change was coming after resolving the conflicts for this pull request. Nithesh has added this for the checkout expiry date feature. So it is taken from dev branch only